examples/all/manifests/thanos-compactor-deployment.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: thanos-compactor
+  name: thanos-compactor
+  namespace: monitoring
+spec:
+  replicas: 1
```

Comment:
I think we need to ensure there is only one active instance. We should ensure this in the update strategy.